package.json

```
@@ -31,10 +31,11 @@
     "cssnano": "4.1.10",
     "del": "6.0.0",
     "easymde": "2.10.2-360.0",
+    "gifsicle": "^5.1.0",
     "gulp": "4.0.2",
     "gulp-concat": "2.6.1",
     "gulp-if": "3.0.0",
-    "gulp-imagemin": "7.1.0",
+    "gulp-imagemin": "^7.1.0",
```

Version fixe pour ne pas avoir de soucis
```suggestion
    "gulp-imagemin": "7.1.0",
```